### PR TITLE
Chore: Fix npm vulnerability

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

There is a vulnerability when running `npm audit`. Not important, but let's keep dependencies clean of vulnerability.

# Changes

## Automatic changes

* Changes in package-lock.json after running `npm audit fix`.

# Tests

Still passing.

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3955** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/3955?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/3955?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd @peterpeterparker please review

File: CHANGELOG-Nns-Dapp-unreleased.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ proposal is successful, the changes it released will be moved from this file to
 
 #### Security
 
+* Fix dependency vulnerability.
```
Comment (peterpeterparker) [inline CHANGELOG-Nns-Dapp-unreleased.md:32]: That sound way to scary, can you be more precise?

Like "Bump css-tools dev dependency to patch xyz"

**Review by peterpeterparker — APPROVED** [review]
Thanks

**Pull request merged**